docs/support/tutorials/index.md

```diff
@@ -57,10 +57,12 @@
 * [Starting with parallel R](parallel-r.md)
 
 ## Geoinformatics
+* [Geocomputing in CSC computing environment](https://research.csc.fi/geocomputing)
 * [Earth Observation (EO) guide](gis/eo_guide.md)
 * [Using geospatial files directly from cloud, inc Allas](gis/gdal_cloud.md)
 * [GDAL tutorial](gis/gdal.md)
 * [Virtual rasters](gis/virtual-rasters.md)
+* For software specific tutorials, see [the respective software pages](../../apps/by_discipline.md#geosciences)
```

Comment:
```suggestion
* For software specific tutorials and examples, see [the respective software pages](../../apps/by_discipline.md#geosciences)
```
Not every page has a tutorial?

Reply:
Not every page has them, but the main ones (Python and R) have rather nice listings there also...